### ISSUE TITLE
LND peers fails to send revocation ack, channels stuck offline

### ISSUE THREAD

**Issue by viaj3ro** [lightningnetwork/lnd#6277]
I'm running eclair 0.6.2

I have several channels with the bluewallet node: 
682798x821x1
698337x1000x1
715719x2221x1

All channels are offline and upon reconnecting, I receive this:

```
2022-01-30 23:11:05,863 WARN  f.a.e.channel.Channel n:037cc5f9f1da20ac0d60e83989729a204a33cc2d8e80438969fadf35c1c5f1233b c:af5de2f0ee565aa13c84a42bfbbc26513933e27fc7b1fbde0314ae62f107d4eb - waited for too long for a revocation to remoteCommitNumber=12581, disconnecting
2022-01-30 23:11:05,863 INFO  fr.acinq.eclair.io.Peer CON n:037cc5f9f1da20ac0d60e83989729a204a33cc2d8e80438969fadf35c1c5f1233b - disconnecting
2022-01-30 23:11:05,863 INFO  f.a.e.io.PeerConnection CON n:037cc5f9f1da20ac0d60e83989729a204a33cc2d8e80438969fadf35c1c5f1233b - stopping with reason=UserRequest
2022-01-30 23:11:05,863 INFO  fr.acinq.eclair.io.Peer CON n:037cc5f9f1da20ac0d60e83989729a204a33cc2d8e80438969fadf35c1c5f1233b - connection lost
2022-01-30 23:11:05,863 INFO  f.a.e.c.TransportHandler CON n:037cc5f9f1da20ac0d60e83989729a204a33cc2d8e80438969fadf35c1c5f1233b - stopping (unackedReceived= unackedSent=None)
2022-01-30 23:11:05,863 INFO  f.a.e.i.ReconnectionTask CON n:037cc5f9f1da20ac0d60e83989729a204a33cc2d8e80438969fadf35c1c5f1233b - peer got disconnected shortly after connection was established, next reconnection in 3600 seconds
```

the issue seems to be this: `waited for too long for a revocation to remoteCommitNumber=12581, disconnecting`

I've raised the [issue ](https://github.com/BlueWallet/BlueWallet/issues/4459)in the bluewallet repo, but according to them, it got nothing to do with their node specifically and is an LND issue instead.

**Comment by Crypt-iQ** [comment]
Ah, my bad, it seems like lnd is not sending back a revoke_and_ack here. That's pretty odd, I think we'd need more logs probably from the lnd side to diagnose this

**Comment by viaj3ro** [comment]
I asked them to provide logs. I hope they respond.

**Comment by Crypt-iQ** [comment]
cc @t-bast what deadline for receiving revoke_and_ack does eclair use?

**Comment by t-bast** [comment]
This is configurable by node operators, the default is 20 seconds: https://github.com/ACINQ/eclair/blob/be78e0ca578726015bd32e45bd0310415061843c/eclair-core/src/main/resources/reference.conf#L113

If eclair is expecting you to send a `revoke_and_ack`, it will disconnect after this timeout if it doesn't receive anything, hoping that you'll send it again after `channel_reestablish`.

**Comment by Crypt-iQ** [comment]
Thanks @t-bast. 

@viaj3ro After eclair reconnects to lnd, do the logs show the issue persisting for the channel?

**Comment by viaj3ro** [comment]
yes, issue still persists for the two remaining channels. One has been force closed in the meantime (apparently due to a timed out HTLC)

**Comment by DerEwige** [comment]
Had the same problem today with this node:
https://amboss.space/node/035b710d2c8d25993ddcbe031c0fee5434d5e6fb7a6ed5d438404daa99199537b8

Increased my timeout to 40s and finally managed to connect and close the channel.
Looks like the node runs on very low power machine and just can not keep up with the load
